addons/asset-conflict-dialog/style.css

```diff
@@ -0,0 +1,49 @@
+.conflictDialog {
+  box-sizing: border-box;
+  width: 600px;
+  max-height: 80vh;
+  max-width: 85%;
+  margin-top: 12vh;
+  overflow-y: auto;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.conflictDialog-content {
+  padding: 1.5rem 2.25rem;
+}
+
+.conflictDialog-content > p {
+  margin-top: 0;
+}
+
+.conflictDialog-actions {
+  display: flex;
+  flex-direction: row;
+  margin-bottom: 1rem;
+  gap: 0.5rem;
+}
+
+.conflictDialog-button {
```

Comment:
Using `addon.tab.scratchClass` you should be able to apply Scratch's button classes which are automatically compatible with editor dark mode.